openmrs-19/src/main/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceImpl.java

```diff
@@ -42,14 +44,16 @@
     private OpenMRSProviderService providerService;
 
     @Override
-    public void createEncounter(DateTime encounterDatetime, String encounterType, String locationName, String patientUuid, String providerUuid) {
+    public void createEncounter(DateTime encounterDatetime, String encounterType, String locationName, String patientUuid, String providerUuid, Map<String, String> observations) {
         Location location = getLocationByName(locationName);
         Patient patient = patientService.getPatientByUuid(null, patientUuid);
         Provider provider = providerService.getProviderByUuid(null, providerUuid);
 
+        List<Observation> observationList = convertObservationMapToList(observations, encounterDatetime);
```

Comment:
Observations can be null, because it is not required field in action. So in method convertObservationMapToList will be NPE